layouts/shortcodes/featured/slider.html

```diff
@@ -1,47 +1,100 @@
 {{ $id := partial "helper/new-id" . }}
```

Comment:
slider and featured/slider almost identical except for the container class. Can we have them both call say partials/tw/slider.html and pass in a .containerClass or something like that to reduce the duplication?